it-IT.Rmd

```diff
@@ -97,18 +96,29 @@ Il nostro gruppo sarà felice di aiutarti e di offrirti il supporto necessario (
 
 Diamo valore alla tua presenza e partecipazione agli eventi di Bioconductor e alla nostra comunità.
 
+#### _Mediatore/Mediatrice_
+
+Capiamo che tu possa volere una persona non collegata a Bioconductor che superviosioni la risoluzione del tuo conflitto. Per facilitare ciò, la Commissione del Codice di Comportamento nominerà un/una mediatore/mediatrice da una comunità open-source differente rispetto ai membri del progetto di Bioconductor. Mentre loro non hanno il potere di stabilire, cambiare o impostare decisione prese dalla Commissione del Codice di Comportamento, possono assistere i membri della comunità offrendo loro consultazione, mediazione, istruzione e riferimenti ad altre risorse. Il/La mediatore/mediatrice possono essere contattati direttamente via mail per assicurare un processo confidenziale, imparziale, informale e indipendente.
+
+#### _Procedura dell'incidente_
+
+La Commissione del Codice di Comportamento segue una procedura standard per trattare i conflitti.
+Questa procedura è spigata brevemente [qui](https://bioconductor.github.io/CodeOfConduct/incident_response.html). Le procedure d'incidente sono basate sul libro "How to Respond to Code of Conduct Reports" di Valerie Aurora e Mary Gardiner.
```

Comment:
Can we/shall we link to this book - valid also backwards for the english version?

Reply:
mmm I don't know where this book come from, maybe we can open an issue on the repo.